modules/troubleshooting-network-observability-controller-manager-pod-out-of-memory.adoc

```diff
@@ -6,26 +6,31 @@
 [id="controller-manager-pod-runs-out-of-memory_{context}"]
 = Network Observability controller manager pod runs out of memory
 
-You can increase memory limits for the Network Observability operator by patching the Cluster Service Version (CSV), where Network Observability controller manager pod runs out of memory.
+You can increase memory limits for the Network Observability operator by editing the `Subscription` object
 
 .Procedure
 
-. Run the following command to patch the CSV:
+* Increase memory limits to `800Mi` by editing the `Subscription` object:
```

Comment:
Maybe make it clear that 800Mi is just an example, not a hard requirement?

Reply:
Agreed, maybe we can link a reference to the Resource Recommendations

Reply:
Good idea to link there. I will add that.

Reply:
Just added that link.